### PR TITLE
[#21] Connected Android module with naver webtoon.

### LINKED ISSUE #21: Connect Android module with Naver Webtoon services
- Crawl the Naver Webtoon Content
- Directly queries the Main Webtoons, Best Challenge webtoons through the url.
- Able to list the queried thumbnail images and the episode preview in the first page.

### PR DESCRIPTION
**Pull request by harry2636**
Fixes #21 
- Able to crawl the Naver Webtoon Content including main webtoons, best challenge webtoons and challenge webtoons.
- Able to query the webtoon infos from the Android application.

### CODE REVIEW

File: Android/toonDra/app/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -1,6 +1,7 @@
 <?xml version="1.0" encoding="utf-8"?>
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
     package="edu.kaist.mskers.toondra">
+    <uses-permission android:name="android.permission.INTERNET" />
```
Comment (dansuh17) [inline Android/toonDra/app/src/main/AndroidManifest.xml:4]: unnecessary blank line

**Review by dansuh17 — CHANGES_REQUESTED** [review]

**Comment by dansuh17** [comment]
just to make sure - this is only an importing of files for webtoon crawling, right? no representation on mainactivity yet?

File: Android/toonDra/app/src/main/AndroidManifest.xml (hunk 1 above)
Comment (harry2636) [reply Android/toonDra/app/src/main/AndroidManifest.xml:4]: Fixed it.

**Comment by harry2636** [comment]
@deNsuh Yes, it is. there is no representation on main activity yet. Code for main activity will be committed in another pull request.

Fixed requested points.

**Comment by dansuh17** [comment]
LGTM

**Comment by heewoonam** [comment]
LGTM

**Pull request merged**